seewav.py

```diff
@@ -293,8 +329,11 @@ def main():
                   oversample=args.oversample,
                   time=args.time,
                   fg_color=args.color,
+                  fg_color2=args.color2,
                   bg_color=[1. * bool(args.white)] * 3,
-                  size=(args.width, args.height))
+                  size=(args.width, args.height),
+                  stereo=args.stereo,
+                  )
```

Comment:
nit: closing parenthesis on the same line as last arg, no trailing ','. Some programmers will tell you its bad for git blame + easily adding params, but I like it better with the more compact style.